.github/workflows/security.yml

```diff
@@ -29,3 +29,7 @@ jobs:
     steps:
       - name: Scan for vulnerabilities in go code
         uses: golang/govulncheck-action@v0.2.0
+        with:
+          go-version-input: 1.20.6
```

Comment:
For some reason I need to specify this to be 1.20.6, otherwise it resolve to 1.20.5 even tho we set check latest to true and cache false. 🤷🏻